cmd/sync.go

```diff
@@ -313,8 +314,8 @@ func doSync(c *cli.Context) error {
 	go func() { _ = http.ListenAndServe(fmt.Sprintf("127.0.0.1:%d", config.HTTPPort), nil) }()
 
 	// Windows support `\` and `/` as its separator, Unix only use `/`
-	srcURL := strings.Replace(c.Args().Get(0), "\\", "/", -1)
-	dstURL := strings.Replace(c.Args().Get(1), "\\", "/", -1)
+	srcURL := c.Args().Get(0)
```

Comment:
```suggestion
        srcURL := c.Args().Get(0)
	dstURL := c.Args().Get(1)
	if runtime.GOOS == "windows" && !strings.Contains(srcURL, "://") && isFilePath(srcURL) {
		srcURL = strings.Replace(srcURL, "\\", "/", -1)
	}
	if runtime.GOOS == "windows" && !strings.Contains(dstURL, "://") && isFilePath(dstURL) {
		dstURL = strings.Replace(dstURL, "\\", "/", -1)
	}
```
Is this better ? `strings.HasSuffix(srcURL, "/") != strings.HasSuffix(dstURL, "/") ` should execute  after path transform .

Reply:
ok